templates/admin/perftrace.tmpl

```diff
@@ -0,0 +1,13 @@
+{{template "admin/layout_head" (dict "ctxData" . "pageClass" "admin monitor")}}
+
+<div class="admin-setting-content">
+	{{template "admin/trace_tabs" .}}
+
+	{{range $record := .PerfTraceRecords}}
+	<div class="ui segment tw-w-full tw-overflow-scroll">
```

Comment:
```suggestion
	<div class="ui segment tw-w-full tw-overflow-auto">
```

Don't show scrollbar unless necessary.

Reply:
For > 99% cases, the scroll bar is always there.

Reply:
So, why force it to display?

Reply:
Applied your suggestion.

Actually, this page is just for developers only, the UI is quite simple and crude. End users won't see it. So fine-tuning it or not, it doesn't really affect end users.